packages/cactus-plugin-satp-hermes/src/main/typescript/core/stage-services/client/stage0-client-service.ts

```diff
@@ -109,6 +117,20 @@ export class Stage0ClientService extends SATPService {
       getHash(newSessionRequestMessage),
     );
 
+    await storeLog(
```

Comment:
the procedure is good, however, the operation types are wrongly put. there are only 4 operations, and each action/function should have at least 3 log entries: init, exec, and done or fail. Please see carefully the examples 3.1 and 3.2 at https://ietf-satp.github.io/draft-belchior-satp-gateway-recovery/draft-belchior-satp-gateway-recovery.html

Reply:
also, please change name of the function from storeLog to persistLogEntry